### ISSUE TITLE
Add "JGI - metagenomics - long read" class to submission schema

### ISSUE THREAD

**Issue by bmeluch** [microbiomedata/submission-schema#168]
Due in Dev Feb 23rd

Part of the effort to make the submission portal accommodate JGI user projects. 

The current "JGI MG" tab was created based on short read data. Long read data will use the same slots, with the difference being that [dna_absorb1](https://microbiomedata.github.io/nmdc-schema/dna_absorb1/) and [dna_absorb2](https://microbiomedata.github.io/nmdc-schema/dna_absorb2/) are required. (See https://github.com/microbiomedata/issues/issues/433)

This will require creating a new JGI metagenomics long reads class and submission portal tab (https://github.com/microbiomedata/issues/issues/413#issuecomment-1847996488).

**Comment by turbomam** [comment]
Hi @bmeluch 

Can you say more about the difference between a "tab" and a class? Where could I find "JGI MG" in the submission schema? I did the following search and I don't think I'm finding anything with that name.

https://github.com/search?q=repo%3Amicrobiomedata%2Fsubmission-schema+JGI+MG&type=code

"JGI MG - long reads" would be an unusual name for a class

**Comment by mslarae13** [comment]
@bmeluch are you currently working on this, or waiting on something?

**Comment by mslarae13** [comment]
@pkalita-lbl 
I checked with @bmeluch. She feels like she's done about all she can here. Would you have some time to either take this on and wrap it up or help guide her to close this out?

We think the remaining things to happen are

- [ ] Update the UI to show the additional / update the metagenomics- long and -short
- [ ] Update version / prod to show the 2 interface tabs

**Comment by mslarae13** [comment]
Technically, schema is updated. UI needs updated. Should this be a separate issue?

**Comment by pkalita-lbl** [comment]
Yeah I can take over from here. Don't worry about a new issue for right now I'll make one for myself if needed. I'll add myself as an assignee and add this to the next sprint so I don't lose track of it.

**Comment by mslarae13** [comment]
Need confirmation on additional analysis type. Then into submission schema. 
Rescheduled for March 8th in dev

**Comment by pkalita-lbl** [comment]
Reopening until the submission portal work is also done.

**Comment by pkalita-lbl** [comment]
Still in progress, awaiting review on [this PR](https://github.com/microbiomedata/nmdc-server/pull/1172) which upgrades the submission-schema version.

**Comment by ssarrafan** [comment]
It looks like there is a validation issue on the linked PR. Moving this to the next sprint.  
@bmeluch @pkalita-lbl  if this isn't high priority we can move it to the post-GSP sprint. Let me know.

**Comment by pkalita-lbl** [comment]
Just waiting on final code review here. Moving to next sprint in case it doesn't happen today.
